benchmarks/test_peer_selection.py

```diff
@@ -22,81 +22,78 @@
     absolute_import, unicode_literals, print_function, division
 )
 
-import mock
-import random
+from tornado import gen, ioloop
```

Comment:
This looks like a completely different benchmark. Why are you replacing the existing benchmark with this instead of creating a new one?